modules/card.lua

```diff
@@ -32,7 +32,7 @@ local COMMAND_CARD_CLICK_ARRAY = {
 }
 
 local NP_CARD_CLICK_ARRAY = {
-	Location(1000,220),
+	Location(1000,400),
```

Comment:
I believe @29988122 used `220` instead of `400` for a reason.
My guess is that it prevents the script from accidentally changing targets, because the 1st NP card slightly overlaps the region where the rightmost enemy is.

He should have left a comment about it, though. :sweat_smile: